restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -126,9 +129,23 @@ public void maybeCreateTable(String keyspaceName, String tableName) {
                   String.join(", ", allPathColumnNames)))
           .get();
 
-      dataStore
-          .query(String.format("CREATE INDEX ON \"%s\".\"%s\"(leaf)", keyspaceName, tableName))
-          .get();
+      // After creating the table, it can take up to 2 seconds for permissions cache to be updated,
+      // So retry this first index creation until it succeeds.  The other index creations should not
+      // need this behavior.
+      boolean success = false;
+      while (!success) {
+        try {
+          Uninterruptibles.sleepUninterruptibly(2, TimeUnit.SECONDS);
```

Comment:
You did mention during the standup that this is configurable. Can we extract the configuration value and use it here?
Currently, if someone will change this setting, it will not be reflected in this wait code.

Reply:
It is true that it's configurable in cassandra.yaml; from questions I had asked last week I had thought that stargate doesn't have access to those configs at runtime though?

Either way, I chose to take a different approach that doesn't sleep :) so hopefully I've skirted that issue